packages/truffle-contract/package.json

```diff
@@ -31,6 +31,7 @@
     "bignumber.js": "^7.2.1",
     "ethereumjs-util": "^5.2.0",
     "ethjs-abi": "0.1.8",
+    "legacy-web3": "^0.20.6",
```

Comment:
This strikes me as an anti-pattern. Could this be avoided by making a `truffle-contract-legacy` package instead? Possibly even make it to depend on `truffle-contract` to re-use commonality?